src/Components/Login.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import { signInUser } from '../state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
+import LoginForm from './LoginForm'
+
+class Login extends Component {
+  state = {
+    email: '',
+    password: '',
```

Comment:
Why do you need to save the password in the state?

Reply:
Yes we need it to pass it into the handleLogin function. Is there another way? Are we missing something?